web/styles/subscriptions.css

```diff
@@ -547,7 +631,11 @@ h4.user_group_setting_subsection_title {
     position: relative;
     overflow: auto;
     -webkit-overflow-scrolling: touch;
-    height: calc(100% - 6.0714em); /* 85px at 14px em */
+    /* Calculated from several heights and padding/margin measurements
+    in .list-toggler-container and .stream_filter, with some added
+    fiddling to see what worked best at 12px - 20px font sizes.
+    When we redesign this area we should make this less brittle. */
+    height: calc(100% - 3.4em - 40px);
```

Comment:
This is a bit more accurate than the other one (though it's more eyeballing). I think it would also be fine to leave it as it was if that's simpler.

Reply:
So long as the eyeballing is noted, as it is here, I think this is probably a reasonable improvement pending a full design pass.